api_app/db/repositories/operations.py

```diff
@@ -18,8 +18,13 @@
 
 
 class OperationRepository(BaseRepository):
-    def __init__(self, client: CosmosClient):
-        super().__init__(client, config.STATE_STORE_OPERATIONS_CONTAINER)
+    @classmethod
+    async def create(cls, client: CosmosClient):
+        cls = OperationRepository()
+        base_repo = await super().create(client, config.STATE_STORE_OPERATIONS_CONTAINER)
+        cls._container = base_repo._container
```

Comment:
In case you remove the initialization of the base class in the factory method as commented above,
you should remove those lines as well, and drop the `base_repo` local variable .
```
cls._container = base_repo._container
cls._client = base_repo._client
```
same goes to all repositories.

Reply:
Fixed